### ISSUE TITLE
MPEG-TS video gives a black screen when using ffmpeg mp3 extension

### ISSUE THREAD

**Issue by tpuserhp** [google/ExoPlayer#10855]
Unfortunately we can't answer all questions. Unclear questions or questions with
insufficient information may not get attention.

Before filing a question:
-------------------------

- Ask general Android development questions on Stack Overflow
- Search existing issues, including issues that are closed
  https://github.com/google/ExoPlayer/issues?q=is%3Aissue
- Consult our developer website (https://exoplayer.dev/) and Javadoc
  (https://exoplayer.dev/doc/reference/)

When filing a question:
-------------------------

Describe your question in detail.

In case your question refers to a problem you are seeing in your app:

- Output of running `$ adb bugreport` in the console

In case your question is related to a piece of media:

- URI to test content
- For protected content:
  - DRM scheme and license server URL
  - Authentication HTTP headers

Don't forget to check supported formats and devices
(https://exoplayer.dev/supported-formats.html).

If there's something you don't want to post publicly, please submit the issue,
then email the link/bug report to dev.exoplayer@gmail.com using a subject in the
format "Issue #1234", where #1234 is your issue number (we don't reply to
emails).

I am using 2.18.2 of Exoplayer. I have been using ffmpeg mp3 extension in my app with no problem for the m3u8 streaming. I 
recompiled the ffmpeg mp3 extension for  2.18.2 ExoPlayer.  Since I don't have a m3u8 stream for testing, I've downloaded a Mpeg TS video for testing. Without the ffmpeg mp3 extension, the video play with no sound. But with the ffmpeg mp3 extension is on, I just get a black screen. The Eventlogger shows the the use of ffmpeg mp3 extension. Also there is a loading event that shows mediaPos has a negative value which could be the cause of the problem.  I also used the demo app with the same result.
When I use VLC or MX Player, the video plays fine. 
It could be the video issue or something I missed. I also found issue #10035 which seems to be related to my issue. 
I will send in the video  for your analysis. Thanks for your help.

**Comment by tpuserhp** [comment]
Video file has been sent for analysis. Thanks.

**Comment by tpuserhp** [comment]
I spent more time on the demo app for the video clip. Once I got the black screen, I touched the screen and I  got the screen with the control buttons. It has -00:01 .00:06 on the bottom of the screen. I pressed the fast backward or fast forward button  to reset the negative time and I can play the video with sound again.  It would be nice if Exoplayer can be enhanced to first start playing the video with time zero instead of a negative time. If not, this ticket can be closed. Thanks.
